backend/app/Http/Controllers/UserLearnWordController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\LearnWordRequest;
+use Illuminate\Http\Request;
+use App\Models\UserLearnWord;
+
+class UserLearnWordController extends Controller
+{
+    public function index(Request $request)
+    {
+        $userid =  $request->user()->id;
```

Comment:
You don't need to pass the User id in Request because you can access the authenticated user by Auth::user();